source/desktop/fastdesk/getting_started_guide/FASTdeskGettingStartedGuideAndroid.md

```diff
@@ -1,7 +1,11 @@
-
 # Getting Started with FASTdesk: Android Edition
-> Compatible with Nougat (Android 7.x), Oreo (Android 8.x) and Pie (Android 9.x).For the most optimum experience, we recommend updating your Android device to the latest Android Software.
 
+```eval_rst
+.. note::
+
+    Compatible with Nougat (Android 7.x), Oreo (Android 8.x) and Pie (Android 9.x).For the most optimum experience, we recommend updating your Android device to the latest Android Software.
+   
+```
```

Comment:
Move this section to below the opening paragraph to prevent ugly search results

Reply:
Is this not compatible with Android 10.x?

Reply:
All isues fixed